other/lru_cache.py

```diff
@@ -114,11 +132,46 @@ def set(self, key: int, value: int) -> None:
             node.val = value
             self.list.add(node)
 
-    def has_key(self, key: int) -> bool:
+    def cache_info(self) -> str:
```

Comment:
```suggestion
    def __str__(self) -> str:
```

Reply:
using __repr__ to use `>>> cache` in doctests